src/main/java/com/swp/board/domain/BoardService.java

```diff
@@ -106,4 +108,16 @@ public List<BoardResponseDto> getMyBoardList(JwtUserDetails userDetails) {
                 .map(board -> BoardResponseDto.of(board))
                 .collect(Collectors.toList());
     }
+
+    @Transactional
+    public List<BoardResponseDto> getFollowingUserBoard(JwtUserDetails userDetails, Pageable pageable) {
+        User user = userRepository.findByProviderAndProviderId(userDetails.getProvider(), userDetails.getUsername())
+                .orElseThrow(() -> new UserNotFoundException("없는 유저입니다"));
+
+        return boardRepository.findAllByUserIn(
+                user.getFollowingList().stream()
+                        .map(Relationship::getToUser)
+                        .collect(Collectors.toList()), pageable
+        ).stream().map(board -> BoardResponseDto.of(board)).collect(Collectors.toList());
```

Comment:
이거 `from()`으로 가기로 하지 않았나요??
```suggestion
        ).stream().map(board -> BoardResponseDto.from(board)).collect(Collectors.toList());
```

Reply:
추가적으로 코드가 조금 긴 것 같은데 `Collectors.toList()` 부분에서 `toList()`를 static import해서 간결하게 유지하는 것도 좋을 것 같아요!

```suggestion
        ).stream().map(board -> BoardResponseDto.of(board)).collect(toList());
```

Reply:
from으로 고쳤던것 같은데 of로 되어 있네요. 수정하겠습니다. 그리고 static import 하겠습니다.